src/main/java/mate/academy/model/FruitTransaction.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private final String operation;
+
+        Operation(String operation) {
+            this.operation = operation;
+        }
+
+        public String getOperation() {
+            return operation;
+        }
+
+        public static Operation get(String operation) {
+            switch (operation) {
+                case "b" : return BALANCE;
+                case "s" : return SUPPLY;
+                case "p" : return PURCHASE;
+                case "r" : return RETURN;
+                default: throw new RuntimeException("Can't find provided operation: " + operation);
```

Comment:
I think it's better to implement this with loop search, so you don't need to add new case construction if you add new operation(if you want to optimize search, think how can you use Map for this purpouse)